### ISSUE TITLE
Empty Sites downloads on staging

### ISSUE THREAD

**Issue by jmithani** [security-force-monitor/sfm-cms#641]
In testing #602, it was discovered that trying to download sites results in empty files (with headers). Locally this does not happen.

**Comment by jmithani** [comment]
Possibly blocked by #610

**Comment by tlongers** [comment]
Because if locations are classed as `point` rather than `node` then the site query would be empty?

**Comment by jeancochrane** [comment]
That's what we're thinking @tlongers. I have a feeling that fixing #610 will fix this, but we can circle back and investigate more if it doesn't.

**Comment by jeancochrane** [comment]
Pushed a quick fix in https://github.com/security-force-monitor/sfm-cms/commit/272868f44e48b87fd76c656bc0856ce7d3414a2b that opens up Sites downloads to Locations that have the `feature_type` `point` in addition to `node`. Once it deploys we should see  entities in the Sites downloads again.

**Comment by tlongers** [comment]
Superseded by changes to Location creation process. Closing.
